test/unit/LevelPlaneTest.js

```diff
@@ -1057,3 +1057,73 @@ test('weak charge: destroy block', t => {
   t.end();
 });
 
+test('conduit activation success: place block', t => {
+  const data = [
+    '','prismarine','prismarine','prismarine','prismarine',
+    'prismarine','','','','prismarine',
+    'prismarine','','conduit','','prismarine',
+    'prismarine','','','','prismarine',
+    'prismarine','prismarine','prismarine','prismarine','prismarine',
+  ];
```

Comment:
Nit: can you format these so the layout of the map is clear? (2 more below)

```js
const data = [
  '',           'prismarine', 'prismarine', 'prismarine', 'prismarine',
  'prismarine', '',           '',           '',           'prismarine',
  'prismarine', '',           'conduit',    '',           'prismarine',
  'prismarine', '',           '',           '',           'prismarine',
  'prismarine', 'prismarine', 'prismarine', 'prismarine', 'prismarine',
];
```

Reply:
Yes sir, certainly can!